collection-scripts/gather_services_status

```diff
@@ -180,6 +183,13 @@ get_octavia_status() {
     run_bg ${BASH_ALIASES[os]} loadbalancer provider list '>' "$OCTAVIA_PATH"/provider_list
 }
 
+# Glance service gathering - task
+get_glance_status() {
+    local GLANCE_PATH="$BASE_COLLECTION_PATH/ctlplane/glance"
+    mkdir -p "$GLANCE_PATH"
+    run_bg ${BASH_ALIASES[os]} image task list '>' "$GLANCE_PATH"/task_list
```

Comment:
I'm ok with this change, which is also useful for troubleshooting purposes.
I'll let @konan-abhi comment before landing this patch, but lgtm.

Reply:
Looks good, better if we change the method name from get_glance_status to get_glance_tasks_status.

Reply:
This is a generic method that is used to collect all useful data, which currently I could only find to be the task list. If you look at get_neutron_status or get_cinder_status, it collects a lot of data from different commands.
Later when OSC supports it, I'm planning to add detailed store information to tell us which glance stores are configured and with which properties.